ui/job-view/details/BugFiler.jsx

```diff
@@ -458,22 +458,16 @@ export class BugFilerClass extends React.Component {
           comment_tags: 'treeherder',
         };
 
-        const bugResp = await create(
+        const { data, failureStatus } = await create(
           getApiUrl('/bugzilla/create_bug/'),
           payload,
         );
-        // const bugResp = await create('http://httpstat.us/404', payload);
-        const data = await bugResp.json();
-        if (bugResp.ok) {
+
+        if (!failureStatus) {
           successCallback(data);
           toggle();
         } else {
-          this.submitFailure(
-            'Treeherder Bug Filer API',
-            bugResp.status,
-            bugResp.statusText,
-            data,
-          );
+          this.submitFailure('Treeherder Bug Filer API', failureStatus, data);
```

Comment:
This looks like it's missing one param: ``statusText``.  This will send in ``data`` in its place.

Reply:
Ah, so the point was to send `data` in place of `statusText` since that's all we have (data and failureStatus) and if the data arg is absent it just wont' append to the `failureString` unless I'm missing something.

Reply:
Oh ok.  As long as that's intended.  :)  I'll test this now.